unit/CMakeLists.txt

```diff
@@ -25,14 +22,29 @@ list(REMOVE_ITEM sources
 
     # Will be built into a separate library and linked
     ${testing_utils}
-    ${java_testing_utils}
+
+    # Java front-end is being remove, so drop all java tests
+    ${java_tests}
```

Comment:
I am confused, unless GitHub is just displaying the commits in the wrong order?

Reply:
This change basically picks up the expanded exclusion list from the previous patch hunk.